lib/src/generate/build_impl.dart

```diff
@@ -168,7 +168,7 @@ class BuildImpl {
 
   /// Asset containing previous asset dependency graph.
   AssetId get _assetGraphId =>
-      new AssetId(_packageGraph.root.name, '.build/asset_graph.json');
+      new AssetId(_packageGraph.root.name, '.dart_tool/build/asset_graph.json');
```

Comment:
DRY this up. Make a const in `lib/src/some_util.dart` or something.

https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)